int32[] memory_full_info
int32 cpu_num
int32 num_threads
int32 pid

float32 memory_percent
float32 cpu_percent

string username
string name
string status

